templates/modules/b2b-account/quotes/email-quote.hypr.live

```diff
@@ -0,0 +1,30 @@
+{% require_script "modules/b2b-account/b2b-contacts" %}
+
+{% extends "modules/common/modal-dialog" %}
+ <div class="modal-header">
+      {% block modal-header %}
+        <button type="button" class="close" aria-hidden="true"  data-dismiss="modal">×</button>
+        <h2 class="modal-title modal-title-text">Email Quote</h2>
+      {% endblock modal-header %}
+      </div>
+{% block modal-body %}
+<div id="quotes-messages">
+    {% include "modules/common/message-bar" with model=model.messages %}
+</div>
+<div class="mz-l-stack-section mz-orderlist" style="clear:none">
+   
+    </br>
+        <div> 
+               <label class="mz-email-label">Email Addresses (Seperate Emails with Comma)</label>
```

Comment:
"Email Addresses (Seperate Emails with Comma)" add this to a labels files. 
In core theme, we have files 'en-US.json' and 'de-DE.json' (german version), add key/value there and use in templates like {{labels.key}}

Reply:
done